plugins/lyricsmania/__init__.py

```diff
@@ -76,8 +76,12 @@ def find_lyrics(self, track):
 
         try:
             lyrics_body = lyrics_html.find_class('lyrics-body')[0]
+            video_elem = lyrics_body.get_element_by_id('video-musictory')
+            first_line = video_elem.tail
             lyrics_body.remove(lyrics_body.get_element_by_id('video-musictory'))
-            lyrics = re.sub(r'^\s+Lyrics to .+', '', lyrics_body.text_content())
+            lyrics = lyrics_body.text_content()
+            lyrics = re.sub(r'^\s+Lyrics to .+', '', lyrics)
```

Comment:
FWIW, I think this line is a no-op (nowadays), because the *Lyrics to ...* element seems to be contained in the parent `<div>` of the `lyrics-body` `<div>` that we process, so we don't actually ever see that string.